wizard/WizardLanguage.qml

```diff
@@ -0,0 +1,253 @@
+                }
+            }
+        }
+
+        Image {
+            id: globe
+            source: "../images/world-flags-globe.png"
+            opacity: 0
+            property bool small: appWindow.width < 700 ? true : false
+            property int size: {
+                if(small){
+                    return 196;
+                } else {
+                    return 312;
+                }
+            }
+            Layout.preferredWidth: size
+            Layout.preferredHeight: size
+            anchors.horizontalCenter: parent.horizontalCenter
+            mipmap: true
+
+            property bool animSlow: false
+            property int animSpeedSlow: 4000
+            property int animSpeedNormal: 120000
+            property real animFrom: 0
+            property real animTo: 360
+
+            Rectangle {
+                visible: !globe.small
+                anchors.left: parent.left
+                anchors.top: parent.top
+                anchors.leftMargin: 117 * scaleRatio
+                anchors.topMargin: 71 * scaleRatio
+                width: 36 * scaleRatio
+                height: 40 * scaleRatio
+                color: "transparent"
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        anim.stop();
+                        globe.animFrom = globe.rotation;
+                        globe.animTo = globe.animFrom + 360;
+                        anim.duration = globe.animSlow ? globe.animSpeedNormal : globe.animSpeedSlow;
+                        globe.animSlow = !globe.animSlow;
+                        anim.start();
+                    }
+                }
+            }
+
+            Behavior on opacity {
+                NumberAnimation {
+                    duration: 450;
+                    easing.type: Easing.InCubic;
+                }
+            }
+
+            RotationAnimation on rotation {
+                id: anim
+                loops: Animation.Infinite
+                from: globe.animFrom
+                to: globe.animTo
+                duration: globe.animSpeedNormal
+            }
+        }
+
+        GridLayout {
+            id: buttonsGrid
+            opacity: 0
+            columns: isMobile ? 1 : 2
+            anchors.horizontalCenter: parent.horizontalCenter
+            Layout.topMargin: 20 * scaleRatio
+            Layout.fillWidth: true
+            columnSpacing: 20 * scaleRatio
+
+            MoneroComponents.StandardButton {
+                id: idChangeLang
+                Layout.minimumWidth: 150 * scaleRatio
+                text: "Language"
+
+                onClicked: {
+                    languageSidebar.open();
+                }
+            }
+
+            MoneroComponents.StandardButton {
+                id: btnContinue
+                Layout.minimumWidth: 150 * scaleRatio
+                text: "Continue"
+
+                onClicked: {
```

Comment:
Is this extra click on the continue button required. If I click a language, that should be good to move on.

Reply:
I think this is pretty confusing in terms of UX. Clicking language in the sidebar should not do anything more than (dynamically) switching the language it is currently displaying.

In addition, I think the complexity of code needed does not justify the goal.